hs_geo_raster_resource/models.py

```diff
@@ -65,6 +65,7 @@ def create(cls, **kwargs):
                 if value_item not in value_arg_dict:
                     raise ValidationError("For coverage of type 'box' values for one or more "
                                           "bounding box limits or 'units' is missing.")
+                    # TODO: Pabitra - validate also the values of the bounding box
```

Comment:
@pkdash Just curious, thought the bounding box is extracted from the tif file, so what is the idea for validating the values of the bounding box in your TODO comment?